docker-compose-superset.yml

```diff
@@ -0,0 +1,90 @@
+services:
+
+  superset:
+    depends_on:
+      redis:
+        condition: service_started
+      postgresql:
+        condition: service_started
+      superset-init:
+        condition: service_completed_successfully
+    environment: &superset-env
+      - DATABASE_HOST=${POSTGRES_DB_HOST}
+      - DATABASE_DB=${SUPERSET_DB}
+      - DATABASE_USER=${SUPERSET_DB_USER}
+      - DATABASE_PASSWORD=${SUPERSET_DB_PASSWORD}
+      - SECRET_KEY=${SECRET_KEY}
+      - ADMIN_PASSWORD=${SUPERSET_ADMIN_PASSWORD}
+      - ANALYTICS_DB_PASSWORD=${ANALYTICS_DB_PASSWORD}
+    image: &superset-image amancevice/superset:1.5.1
```

Comment:
Does the `docker-compose-superset.yml` file get inherited by ozone-pro? Since we've already upgraded Superset to `v2.x` for the `ozone-pro` project, I believe we should make the upgrade/changes here as well.

Reply:
I've upgraded this.